deploy/twingate-operator/crds/twingate.com.twingateconnector.yaml

```diff
@@ -101,6 +101,18 @@ spec:
             status:
               type: object
               x-kubernetes-preserve-unknown-fields: true
+      additionalPrinterColumns:
+        - name: ID
+          type: string
+          description: "The ID of the connector."
+          jsonPath: .spec.id
+        - name: Display Name
+          type: string
+          description: "Name of the connector."
+          jsonPath: .spec.name
+        - jsonPath: .metadata.creationTimestamp
+          name: Age
+          type: date
```

Comment:
nit: the order of the keys are not like the 2 above. Also missing `description`? 🙈

Reply:
I think its self described